nautobot_ssot/jobs/base.py

```diff
@@ -268,7 +268,7 @@ def __init__(self):
         # Default diffsync flags. You can overwrite them at any time.
         self.diffsync_flags = DiffSyncFlags.CONTINUE_ON_FAILURE | DiffSyncFlags.LOG_UNCHANGED_RECORDS
 
-    def as_form(self, data=None, files=None, initial=None):
+    def as_form(self, data=None, files=None, initial=None, approval_view=False):
         """Render this instance as a Django form for user inputs, including a "Dry run" field."""
         form = super().as_form(data=data, files=files, initial=initial)
```

Comment:
Tricky here as `approval_view` is only a parameter in nautobot 1.2 and later - perhaps we need something like:

```suggestion
    from nautobot import __version__ as nautobot_version
    from packaging.version import Version
    if Version(nautobot_version) < Version("1.2"):
        form = super().as_form(data=data, files=files, initial=initial)
    else:
        form = super().as_form(data=data, files=files, initial=initial, approval_view=approval_view)
```